src/main/entities/Researcher.java

```diff
@@ -0,0 +1,66 @@
+package entities;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Researcher extends User {
+
+    /**
+     * A list of Study objects that the this researcher belongs to (works in).
+     */
+    private List<Study> listStudies = new ArrayList<Study>();
+
+    /**
+     * @param username the username of this Researcher.
+     * @param name     the name of this Researcher.
+     */
+    public Researcher(String username, String name) {
+        super(username, name);
+    }
+
+    /**
+     * @return the list of Study objects this Researcher belongs to, i.e. works in.
+     */
+    public List<Study> getListStudies() {
+        return listStudies;
+    }
+
+    /**
+     * @param study
+     * @return true if this Researcher is part of study.
+     */
+    public boolean listStudiesContains(Study study) {
+        return listStudies.contains(study);
+    }
+
+    /**
+     * Enroll this researcher to study
+     *
+     * @param study
+     * @return true iff researcher was enrolled in study.
+     */
+    public boolean addToListStudies(Study study) {
+        if (this.listStudiesContains(study)) {
+            return false;
+        }
+        listStudies.add(study);
+        return true;
+    }
+
+    /**
+     * Removes the a Study from this researcher's list of Studies. Either because the study was close, or the Researcher
+     * is no longer part of the Study.
+     *
+     * @param study
+     * @return
+     */
+    public boolean removeFromListStudies(Study study) {
+        if (this.listStudiesContains(study)) {
+            listStudies.remove(study);
+            return true;
+        }
+        return false;
+    }
+

```

Comment:
If we remove/add a researcher from the study, shouldn't we update the attribute 'researchers' in class Study as well?

Reply:
Yes! but I believe we have a Use Case that calls on this method when a researcher is removed. That same Use Case will have to call on the Study object this researcher belongs to, to remove this researcher from its attribute (i.e. the Study class should have a removeResearcher method. So a researcher object doesn't directly call a method of a study object.

Reply:
got it,thanks!